<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>591</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>127</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="styleSheet">
        <string notr="true">font-weight : bold;</string>
       </property>
       <property name="text">
        <string>Charger Video</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Desktop/stage/Stage_Orleans/images/browser.ico</normaloff>../Desktop/stage/Stage_Orleans/images/browser.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectToCamera">
       <property name="styleSheet">
        <string notr="true">font-weight : bold;</string>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Desktop/stage/Stage_Orleans/images/camera.jpg</normaloff>../Desktop/stage/Stage_Orleans/images/camera.jpg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>300</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="playStopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight : bold;</string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Documents/images/go.png</normaloff>../Documents/images/go.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveVideo">
       <property name="styleSheet">
        <string notr="true">font-weight : bold;</string>
       </property>
       <property name="text">
        <string>Save VIdeo</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Desktop/stage/Stage_Orleans/images/save.jpeg</normaloff>../Desktop/stage/Stage_Orleans/images/save.jpeg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">background-color: green; color:white;</string>
       </property>
       <property name="text">
        <string>Bouton 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="styleSheet">
        <string notr="true">background-color:red;color:white;</string>
       </property>
       <property name="text">
        <string>Bouton 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fichie&amp;r</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Editio&amp;n</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_connectToCamera()</slot>
 </slots>
</ui>
